specification/networkanalytics/resource-manager/Microsoft.NetworkAnalytics/stable/2023-11-15/networkanalytics.json

```diff
@@ -0,0 +1,2571 @@
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "DataTypes_GenerateStorageContainerSasToken_MaximumSet_Gen": {
+            "$ref": "./examples/DataTypes_GenerateStorageContainerSasToken_MaximumSet_Gen.json"
+          },
+          "DataTypes_GenerateStorageContainerSasToken_MaximumSet_Gen - generated by [MinimumSet] rule_MinimumSet_Gen": {
+            "$ref": "./examples/DataTypes_GenerateStorageContainerSasToken_MinimumSet_Gen.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.NetworkAnalytics/dataProducts/{dataProductName}/generateStorageAccountSasToken": {
+      "post": {
+        "operationId": "DataProducts_GenerateStorageAccountSasToken",
+        "tags": [
+          "DataProducts"
+        ],
+        "description": "Generate sas token for storage account.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "dataProductName",
+            "in": "path",
+            "description": "The data product resource name",
+            "required": true,
+            "type": "string",
+            "minLength": 3,
+            "maxLength": 63,
+            "pattern": "^[a-z][a-z0-9]*$"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "description": "The content of the action request",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/AccountSas"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "ARM operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/AccountSasToken"
+            }
+          },
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "DataProducts_GenerateStorageAccountSasToken_MaximumSet_Gen": {
+            "$ref": "./examples/DataProducts_GenerateStorageAccountSasToken_MaximumSet_Gen.json"
+          },
+          "DataProducts_GenerateStorageAccountSasToken_MaximumSet_Gen - generated by [MinimumSet] rule_MinimumSet_Gen": {
```

Comment:
>generated by [ [](http://example.com/codeflow?start=72&length=14)

you won't want the 'generated by' to show up in your example documents

Reply:
This is cloned from private repo. We will first make required change in the private repo and make a fix in public repo in the next release.